### PR TITLE
Update FundMe.sol

### PR DESCRIPTION
**Pull request by aviralgoel**
added better comments

### CODE REVIEW

**Review by DarshGupta1910 — COMMENTED** [review]
Looks Good to me.

File: FundMe.sol [hunk 1]
```diff
@@ -31,32 +44,42 @@ contract FundMe {
...
     
     // 1000000000
     function getConversionRate(uint256 ethAmount) public view returns (uint256){
         uint256 ethPrice = getPrice();
         uint256 ethAmountInUsd = (ethPrice * ethAmount) / 1000000000000000000;
+        // the actual ETH/USD conversation rate, after adjusting the extra 0s.
         return ethAmountInUsd;
     }
     
+    //modifier: https://medium.com/coinmonks/solidity-tutorial-all-about-modifiers-a86cf81c14cb#:~:text=The%20Solidity%20documentation%20define%20a%20modifier%20as%20follow%3A,of%20the%20function%20to%20which%20it%20is%20attached.
```
Comment (PatrickAlphaC) [inline FundMe.sol]:
```suggestion
    //modifier: https://medium.com/coinmonks/solidity-tutorial-all-about-modifiers-a86cf81c14cb
```

**Review by PatrickAlphaC — CHANGES_REQUESTED** [review]
Just one thing and then it's good!

**Comment by PatrickAlphaC** [comment]
Thank you!

**Pull request merged**